### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by isf63**

### CODE REVIEW

**Comment by stefonarch** [comment]
Maybe we could mention also wayland compatibility @tsujan ?

**Review by luis-pereira — APPROVED** [review]

**Comment by tsujan** [comment]
> Maybe we could mention also wayland compatibility

Yes, somehow, especially because ``lxqt-globalkeys` is for X11, while lxqt-runner works under Wayland.

**Comment by isf63** [comment]
So lxqt-globalkeys is a complement to X11 WMs, but not Wayland compositors? If that's the case, do Wayland compositors have any mechanism to assign hotkeys?

**Comment by tsujan** [comment]
> do Wayland compositors have any mechanism to assign hotkeys?

Yes. For example, I use LabWC and have assigned `W-F1` (= Meta+F1) to lxqt-runner.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Technically it consists of a single binary `lxqt-runner`. The binary is run in a
 
 ### Compiling source code
 
-Runtime dependencies are muparser, KWindowSystem, menu-cache and [lxqt-globalkeys](https://github.com/lxqt/lxqt-globalkeys).  
+Runtime dependencies are [lxqt-globalkeys](https://github.com/lxqt/lxqt-globalkeys), muparser, KWindowSystem, and optionally menu-cache.
```
Comment (stefonarch) [inline README.md:17]: I'd add something like:

**Note**: As `lxqt-globalkeysd` is not supported yet on wayland a shortcut for `lxqt-runner` has to be added in the compositor settings.

**Comment by tsujan** [comment]
Why removing double spaces at ends of lines? They have meaning in markdown (FeatherPad doesn't remove them, for example).

**Comment by isf63** [comment]
My bad! I incorrectly assumed they were artifacts.

**Comment by tsujan** [comment]
I said that in general. Although they may have no effect in this case, they can serve as line separators in markdown.

**Review by tsujan — APPROVED** [review]
Thanks!

**Pull request merged**